coalib/bears/BEAR_KIND.py

```diff
@@ -1,3 +1,3 @@
 from coalib.misc.Enum import enum
 
-BEAR_KIND = enum('LOCAL', 'GLOBAL')
+BEAR_KIND = enum('LOCAL', 'GLOBAL', 'DEFAULT')
```

Comment:
aaahh wait wait
what shall `DEFAULT`-kind mean?

I meant you shall add a kind to the bears defined in the tests :3